jobs/build/advisories/Jenkinsfile

```diff
@@ -219,6 +234,44 @@ node {
                         )
                     }
                     echo "out: ${out}"
+
+                    jira_cmd = """
+                    if command -v yq; then
+                      rm -rf ocp-build-data;
+                      git clone --single-branch --branch openshift-${params.VERSION} git@github.com:openshift/ocp-build-data.git;
+                      cd ocp-build-data;
+                      yq -r '
+                        .advisories
+                        | keys_unsorted[] as \$k
+                        | "* *\\(\$k)*: https://errata.devel.redhat.com/advisory/\\(.[\$k])"
+                      ' group.yml;
+                    fi || true
+                    """
+                    try {
+                        commonlib.shell(
+                            returnStdout: true,
+                            script: jira_cmd
+                        )
+                    } catch (err) {
+                        echo "Ignoring error ${err}"
+                    }
+                }
+            }
+            stage("trigger full build") {
+                try {
+                    if (params.TRIGGER_FULL_BUILD) {
+                        build(
+                            job: "../aos-cd-builds/build%2F${params.VERSION}",
```

Comment:
This rebuilds RPMs and really ends up doing more than we need for this use case. Plus I'm still leaning toward getting rid of the full build jobs.
Could this just trigger the custom build with images only? And only if there is an `arches_override` section...